client_test.go

```diff
@@ -1,18 +1,17 @@
-package flagsmith_test
+package flagsmith
```

Comment:
Are we moving from `flagsmith_test`  to `flagsmith` here to access private method of real-time module? If yes, we should create a different file for that like `analytics_test`